calico/felix/fetcd.py

```diff
@@ -275,17 +291,13 @@ def watch_etcd(self, update_splitter):
                                       response.modifiedIndex) + 1
 
                 if response.action == "delete":
```

Comment:
My version of this change added support for compareAndDelete, expire and the like.  I wanted to add that in so we can remove the code that does a resync when we hit something unexpected.